README.md

```diff
@@ -41,7 +42,43 @@
 - **Node Funding**: Automate funding Bee nodes with ETH and BZZ tokens (Kubernetes optional).
 - **Dynamic Configuration**: Use flexible YAML-based configs for customizable actions.
 
-Beekeeper simplifies managing and testing Bee nodes, whether deployed in Kubernetes or standalone environments.
+Beekeeper simplifies the management and testing of Bee nodes, whether deployed in Kubernetes or standalone environments.
+
+## Quick Start
+
+### Local Development
```

Comment:
Add description that this is used on our CI, and shortly explain how this can be used in local development.